src/stravalib/model.py

```diff
@@ -677,18 +634,6 @@ class name, photo type, and a key identifier."""
         )
 
 
-class ActivityKudos(Athlete):
-    """
-    Represents kudos an athlete received on an activity.
-
-    Notes
-    -----
-    Activity kudos are a subset of athlete properties.
-    """
-
-    pass
-
```

Comment:
This is empty and should be `SummaryAthlete`, so I replaced its use by the latter.